clusterdock/cli.py

```diff
@@ -87,10 +87,13 @@ def main():
     start_parser.add_argument('--always-pull',
                               help="Pull latest images, even if they're available locally",
                               action='store_true')
-    start_parser.add_argument('--namespace',
+    start_parser.add_argument('-n', '--name',
+                              help='Cluster name to use',
+                              metavar='name')
+    start_parser.add_argument('-ns', '--namespace',
```

Comment:
Let's just keep this as just the long argument (eventually, we'll get rid of this altogether and I'm really not a fan of two-letter short names because they're not a POSIX-happy thing).